examples/basics/src/components/Welcome.astro

```diff
@@ -0,0 +1,257 @@
+---
+import astroLogo from '../assets/astro.svg';
+---
+
+<div id="container">
+	<main>
+		<section id="hero">
+			<a href="https://astro.build"
+				><img src={astroLogo.src} width="115" height="48" alt="Astro Homepage" /></a
+			>
+			<h1>
+				To get started, open the <button
+					onclick=`fetch("/__open-in-editor?file=${process.cwd()}/src/pages/index.astro");`
+					title="Click to open in your editor"
+					><span class="sr-only">src/pages</span><code aria-hidden="true"><pre>src/pages</pre></code
+					></button
+				> directory in your project.
```

Comment:
Ah, re-noting my comment on Yan’s suggestion yesterday w/r/t to use of `title`: https://github.com/withastro/astro/pull/12472#discussion_r1847916256

Reply:
I didn't really understand where the `sr-only` element should be?

Reply:
Sorry, I could have provided a code snippet. So, I would probably do this something like this:

```html
<button
	onclick=`fetch("/__open-in-editor?file=${process.cwd()}/src/pages/index.astro");`
>
	<span class="sr-only">Click to open</span>
	<code><pre>src/pages</pre></code>
	<span class="sr-only">in your editor</span>
</button>
```

That way the accessible button label is “Click to open src/pages in your editor” and only “src/pages” is visible.

Now, THAT SAID, there’s still the question of “should it be more obvious to people NOT using a screen reader or other assistive tech that clicking the button opens in the editor?” But maybe that will all no longer be a question depending on whether we’re still doing open-in-editor or not.